src/transport/mod.rs

```diff
@@ -406,6 +407,13 @@ impl<S: BitmapSlice> io::Read for Reader<'_, S> {
     }
 }
 
+/// Safe wrapper for `sysconf(_SC_PAGESIZE)`.
+#[inline(always)]
+pub fn pagesize() -> usize {
+    // Trivially safe
+    unsafe { sysconf(_SC_PAGESIZE) as usize }
+}
```

Comment:
Will libc/musl cache the sysconf page size or will it cause a syscall?
If syscall is involved, should we use lazy_static() to store the value?

Reply:
fixed.